### PR TITLE
NR85999: Nextcloud - Worked on IO page content

### PR DESCRIPTION
**Pull request by dikshab0202**
Ticket: [NR-85999](https://issues.newrelic.com/browse/NR-85999)
Description: Worked on text changes to the NextCloud IO page as mentioned in the [PR1533](https://github.com/newrelic/newrelic-quickstarts/pull/1533).

image

# Summary

A concise description of the changes being introduced. Please review the pre-merge checklist section to validate this pull request is ready for review and merge. **If it is not ready, please mark the pull request as a draft.**

## Pre merge checklist

- [ ] Did you check you NRQL syntax? - Does it work?
- [ ] Did you include an InstallPlan and Documentation reference?
- [ ] Are all documentation links publically accessible?
- [ ] Did you check your descriptive content for [voice and tone](https://docs.newrelic.com/docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic/)? 
- [ ] Did you check your descriptive content for spelling and grammar errors?

### Dashboards 

- [ ] Does the PR contain a screenshot for each of your dashboards?
- [ ] Do your screenshots show data?
- [ ] Has the [sanitization script](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards) been run on each dashboard?

### Alerts

- [ ] Did you check that your alerts actually work?

### CODE REVIEW

**Comment by dikshab0202** [comment]
Hi @paperclypse , 
I worked on the text changes to the NextCloud IO page as mentioned in the [PR1533](https://github.com/newrelic/newrelic-quickstarts/pull/1533). Could you please review and let me know if something missing.
Thank you..

**Review by pkosanam — APPROVED** [review]
looks good.

**Review by paperclypse — DISMISSED** [review]
Looks good!

File: quickstarts/nextcloud/config.yml [hunk 1]
```diff
@@ -2,20 +2,19 @@ id: fe6eb60f-faff-46f9-bf64-666a874ac855
...
-  With New Relic's Nextcloud monitoring system, keep an eye on your Nextcloud in real-time, correlate transactions moving across the application environment and troubleshoot the issue which you are facing.
-  ### Why monitor Nextcloud?
-  Control, monitor and secure all the data of Nextcloud with Infrastructure monitoring agent and Prometheus open metric integration to view a thorough Nextcloud server dashboard that contains a high-level overview of their app's performance. 
-  - Assist teams in creating architectural maps, tracking down all the issues fast, and observing the Apdex of the app.
-  - Maintain a quick throughput and low latency.
-  - Control over the workflow of Nextcloud.
-  - Check the health of the dependencies such as database, application, server.
-  - Give specific results to help with the problems.
+  ## Comprehensive monitoring for your Nextcloud server
+  You can use New Relic's infrastructure agent to monitor the performance of your Nextcloud server in near real-time, correlate transactions move across the app environment, and identify issues to troubleshoot.
```
Comment (brammerl) [inline quickstarts/nextcloud/config.yml:6]:
```suggestion
  You can use New Relic's infrastructure agent to monitor the performance of your Nextcloud server in near real-time, correlate transactions as they move across the app environment, and identify issues to troubleshoot.
```

File: quickstarts/nextcloud/config.yml [hunk 2]
```diff
@@ -2,20 +2,19 @@ id: fe6eb60f-faff-46f9-bf64-666a874ac855
...
+  ### Why monitor your Nextcloud server?
+  Use the power of our infrastructure monitoring agent and the Prometheus open metric integration to control, monitor, and secure the data on your Nextcloud server. Use our dashboards to get a high-level overview of your server's performance. 
+  - See a map of your server's processes to help you track down the source of your issues, fast.
```
Comment (brammerl) [inline quickstarts/nextcloud/config.yml:9]:
```suggestion
  - See a map of your server's processes to help you track down the source of your issues fast.
```

File: quickstarts/nextcloud/config.yml [hunk 3]
```diff
@@ -2,20 +2,19 @@ id: fe6eb60f-faff-46f9-bf64-666a874ac855
...
+  - Use our tools to help you maintain quick throughput and low latency on your server.
+  - Check the health of your database, application, server Nextcloud dependencies.
```
Comment (brammerl) [inline quickstarts/nextcloud/config.yml:11]:
```suggestion
  - Check the health of your database, application, and server Nextcloud dependencies.
```

File: quickstarts/nextcloud/config.yml [hunk 4]
```diff
@@ -2,20 +2,19 @@ id: fe6eb60f-faff-46f9-bf64-666a874ac855
...
+
   ### What’s included?
-  The New Relic Nextcloud monitoring quickstart offers rapid observability:
-  - Alerts (heap memory usage, scrape errors).
-  - Dashboards (uptime, free space in bytes, total active users, heap memory usage and more).
-  - Enables administrators to keep track of the status and operation of a Nextcloud server installation.
+  Use New Relic to monitor and observe your Nextcloud server to:
+  - Be alerted on things like excessive heap memory usage and scrape errors.
+  - Use dashboards to monitor crucial performance metrics, such as uptime, free space in bytes, total active users, and more.
```
Comment (brammerl) [inline quickstarts/nextcloud/config.yml:16]:
```suggestion
  - Use dashboards to monitor crucial performance metrics, such as uptime, free space, total active users, and more.
```

**Comment by brammerl** [comment]
@dikshab0202 left some comments

**Review by aswanson-nr — APPROVED** [review]

**Pull request merged**